WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderSiteHeaderView.java

```diff
@@ -131,13 +130,15 @@ private void showBlogInfo(ReaderBlog blogInfo) {
         }
 
         try {
-            txtFollowCount.setText(
-                    String.format(getContext().getString(R.string.reader_label_follow_count), blogInfo.numSubscribers));
+            txtFollowCount.setText(String.format(
+                            LocaleManager.getSafeLocale(getContext()),
+                            getContext().getString(R.string.reader_label_follow_count),
+                            blogInfo.numSubscribers));
         } catch (ArithmeticException exception) {
-            // See https://github.com/wordpress-mobile/WordPress-Android/issues/5568
```

Comment:
If we do need this try/catch I assume we still need to deal with this bug in which case I think we should keep the comment here.

Reply:
oh yeah good catch. The bug is still relevant, but now it's being properly handled by my `getSafeLocale(...)` method so I can remove this catch altogether.